shiny/_main.py

```diff
@@ -499,10 +499,32 @@ def try_import_module(module: str) -> Optional[types.ModuleType]:
     ),
     help="Choose a template for your new application.",
 )
-def create(template: Optional[str] = None) -> None:
-    from ._template_utils import template_query
+@click.option(
+    "--mode",
+    "-m",
+    type=click.Choice(
+        ["core", "express"],
+        case_sensitive=False,
+    ),
+    help="Do you want to use a Shiny Express template or a Shiny Core template?",
+)
+@click.option(
+    "--github",
+    "-g",
+    help="The github url for the hosted template",
```

Comment:
I'm confused on what "github url" represents... Is it the GitHub website? Raw link? What if it's in a subdirectory of the repo? Zip folder link?

Reply:
Oh yes that's confusing. I changed it to "The GitHub URL of the template sub-directory. For example https://github.com/posit-dev/py-shiny-templates/tree/main/dashboard". The idea is that people will mostly interact with this by viewing templates on the website and seeing a shiny create command, so this should be less of an issue. The survey template will have `shiny create -g https://github.com/posit-dev/py-shiny-templates/tree/main/survey`